obj-keys/challenges-06.test.js

```diff
@@ -131,7 +131,7 @@ const totalCharacters = arr => {
 //
 // Write a function named getHouses that returns an array of the houses in the data set.
 // Use the getFrom function you wrote in challenge 4.
-// ------------------------------------------------------------------------------------------------
+// -----------------------------------------------------------------------------------------------
```

Comment:
your pull request is not reflecting any of your work. make sure you are not working on master